h2o-core/src/main/java/water/api/schemas3/ModelMetricsBinomialGLMGenericV3.java

```diff
@@ -0,0 +1,41 @@
+package water.api.schemas3;
+
+import hex.ModelMetricsBinomialGLMGeneric;
+import water.api.API;
+
+public class ModelMetricsBinomialGLMGenericV3<I extends ModelMetricsBinomialGLMGeneric, S extends ModelMetricsBinomialGLMGenericV3<I, S>>
+        extends ModelMetricsBinomialGenericV3<I, S> {
+
+  @API(help="residual deviance",direction = API.Direction.OUTPUT)
+  public double residual_deviance;
+
+  @API(help="null deviance",direction = API.Direction.OUTPUT)
+  public double null_deviance;
+
+  @API(help="AIC",direction = API.Direction.OUTPUT)
+  public double AIC;
+
+  @API(help="null DOF", direction= API.Direction.OUTPUT)
+  public long null_degrees_of_freedom;
+
```

Comment:
Can you also add AUC, pr_auc here?

Reply:
It's already there. Only in the predecessor object from which the `Generic` variant inherits its fields.